programs/fileio.c

```diff
@@ -1612,7 +1612,8 @@ FIO_compressFilename_internal(FIO_ctx_t* const fCtx,
         U64 const timeLength_ns = UTIL_clockSpanNano(timeStart);
         double const timeLength_s = (double)timeLength_ns / 1000000000;
         double const cpuLoad_pct = (cpuLoad_s / timeLength_s) * 100;
-        DISPLAYLEVEL(4, "%-20s : Completed in %.2f sec  (cpu load : %.0f%%)\n",
+        DISPLAYLEVEL((g_display_prefs.progressSetting == FIO_ps_always ? 1 : 4), 
```

Comment:
This seems separate from the `--progress` notification issue.
This message is more part of the final status.

And I don't believe that this additional information (time spent) belongs to default, which should remain restricted to 1 line.
We may have room for discussion in determining if it's more appropriate for level `3` (`-v`) or `4` (`-vv`),
but it should certainly not be part of `2` (default).

Reply:
This was added per Nick's comment suggesting we add the final status line when using --progress to overwrite the last progress message

Reply:
The final status line is just the line specifying the final size and compression ratio.
Information about timing and speed are considered supplementary (and do constitute an additional line).

Reply:
Ah got it, will update